analysis/src/analyzer.js

```diff
@@ -67,7 +67,31 @@ class AnalyzerRunner {
 
       if (paths.length === 0) {
         analyzer.analyzePackage().then(analysis => {
-          resolve(generateAnalysis(analysis, analyzer.urlResolver, isInPackage));
+          // Get source paths for all features.
+          const features = Array.from(analysis.getFeatures());
+          const paths = features.map((f) => f.sourceRange && f.sourceRange.file);
```

Comment:
this would map to undefined if sourceRange was undefined, so really you should filter. I guess if you're sure it can't be undefined, just use `!`

Reply:
Done - not TS so `!` not needed.